src/ApiCallable.php

```diff
@@ -134,6 +134,40 @@ private static function setLongRunnning($callable, $longRunningDescriptor)
         return $inner;
     }
 
+    private static function callWithoutRequest($callable, $params)
+    {
+        array_shift($params);
+        return call_user_func_array($callable, $params);
+    }
+
+    private static function createGrpcStreamingApiCall($callable, $grpcStreamingDescriptor)
+    {
+        switch ($grpcStreamingDescriptor['grpcStreamingType']) {
+            case 'ClientStreaming':
+                $apiCall = function () use ($callable, $grpcStreamingDescriptor) {
+                    $response = ApiCallable::callWithoutRequest($callable, func_get_args());
+                    return new ClientStream($response, $grpcStreamingDescriptor);
+                };
+                break;
+            case 'ServerStreaming':
+                $apiCall = function () use ($callable, $grpcStreamingDescriptor) {
+                    $response = call_user_func_array($callable, func_get_args());
+                    return new ServerStream($response, $grpcStreamingDescriptor);
+                };
+                break;
+            case 'BidiStreaming':
+                $apiCall = function () use ($callable, $grpcStreamingDescriptor) {
+                    $response = ApiCallable::callWithoutRequest($callable, func_get_args());
+                    return new BidiStream($response, $grpcStreamingDescriptor);
+                };
```

Comment:
No I think this is good - the Page class is already responsible for making calls, so this is reasonable